.vscode/launch.json

```diff
@@ -196,5 +196,24 @@
       "console": "integratedTerminal",
       "internalConsoleOptions": "neverOpen"
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Generate API Doc (courier-js)",
+      "runtimeExecutable": "yarn",
+      "runtimeArgs": [
+        "build-packages",
+        "&&",
+        "yarn",
+        "workspace",
+        "@trycourier/courier-js",
+        "run",
+        "generate-api-doc",
+        "--local"
+      ],
+      "cwd": "${workspaceFolder}",
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen"
+    }
   ]
-}
+}
```

Comment:
nit: the newlines at the end of files are intentional, fwiw https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline